tests/docker/BUILD

```diff
@@ -252,10 +252,10 @@ container_push(
 container_push(
     name = "push_stamped_test",
     format = "Docker",
-    image = ":set_cmd",
+    image = "@io_bazel_rules_docker//testdata:files_in_layer_with_files_base",
     registry = "localhost:5000",
-    repository = "docker/test/{BUILD_USER}",
-    tag = "{BUILD_USER}",
+    repository = "docker/test/{TEST_STAMP}",
```

Comment:
why not use a fixed repository name / tag and avoid having to use print-workspace-status.sh altogether?
i.e., you probably can create a new container_push that is simpler than this one to use for your e2e test.

Reply:
Revert this test to only test the stamp expansion